scenario_player/main.py

```diff
@@ -180,5 +182,121 @@ def reclaim_eth(obj, min_age):
     reclaim_eth(min_age_hours=min_age, **obj)
 
 
+@main.command('pack-logs')
+@click.option(
+    '--target-dir', default=os.environ.get('HOME', './'), show_default=True,
+    help='Target directory to pack logs to. Defaults to your home directory.'
+)
+@click.option(
+    '--scenario-name', default=None, multiple=True,
+    help='Only Pack the specified scenario. If omitted, all latest scenario files are packed. May be specified several times',
+)
+@click.option(
+    '--pack-n-latest', default=1,
+    help='Specify the max num of log history you would like to pack. Defaults to 1.'
+         'Specifying 0 will pack all available logs for a scenario.',
+)
+@click.option(
+    '--raiden-dir', default=os.environ.get('HOME', '.') + '/.raiden',
+    help="Path to the raiden meta data dir. Defaults to ~/.raiden.",
+)
+@click.option('--post-to-rocket', default=True)
+def pack_logs(post_to_rocket, raiden_dir, pack_n_latest, scenario_name, target_dir):
```

Comment:
Nitpick: since `multiple=True` args always get passed as lists it would make sense to call the parameter `scenario_names`. You can to this in click by adding another positional argument that **doesn't** start with a dash.
E.g.:
```python
@click.option("--some-arg", "some_args", multiple=True)
```

Reply:
aha! Cool, thanks!